resources/wiki/Profiles-Security-Score.md

```diff
@@ -0,0 +1,73 @@
+
+| Internet  Explorer | | | | | | | |
+| ------------- | - | - | - | - | - | - | -| 
+|X-XSS-Protection| CSP(basic support) | X-Content-Type-Options| X-Frame-Options(SAMEORIGIn,ALLOW-FROM) | Strict-Transport-Security | Cookie security HttpOnly| Pragma| Cache-Control|  
+| medium | low | medium| low | medium| low| high | high | 
+
+| Edge | | | | | | | |
+| ------------- | - | - | - | - | - | - | -|
+| X-XSS-Protection| CSP | X-Content-Type-Options| X-Frame-Options | Strict-Transport-Security |Cookie security HttpOnly| Pragma| Cache-Control|  
+| medium | low | medium | low | medium | low | high  | high |
+
+| Firefox | | | | | | | | |
+| ------------- | - | - | - | - | - | - | -| -|
+| CSP | X-Content-Type-Options| X-Frame-Options (SAMEORIGIn,ALLOW-FROM) | Strict-Transport-Security | Public Key Pinning| Referrer-Policy|Cookie security HttpOnly|Pragma| Cache-Control|  
+| high | medium | low| medium | low | - | low   | high |high  |
+
+| Safari | | | | | | | |
+| ------------- | - | - | - | - | - | - | -|
+| X-XSS-Protection| CSP | X-Content-Type-Options| X-Frame-Options (SAMEORIGIn,ALLOW-FROM) | Strict-Transport-Security |Cookie security HttpOnly|Pragma| Cache-Control|  
+| medium| low | medium| low| low | low | high | high|
+
+# Mobile Profiles
+
+| Android webview | | | | | | | | | |
+| ------------- | - | - | - | - | - | - | -| -| -|
+| X-XSS-Protection| CSP | X-Content-Type-Options| X-Frame-Options (SAMEORIGIn) | Strict-Transport-Security | Public Key Pinning| Referrer-Policy|Cookie security SameSite| Pragma| Cache-Control|
+| medium | low | medium  | low | medium  | low | - | low| high| high| 
+
+| Chrome for Android | | | | | | | | | | |
+| ------------- | - | - | - | - | - | - | -| -| -| -|
+| X-XSS-Protection| CSP | X-Content-Type-Options| X-Frame-Options (SAMEORIGIn) | Strict-Transport-Security | Public Key Pinning| Referrer-Policy (same-origin, strict-origin, strict-origin-when-cross-origin) |Cookie security HttpOnly|Cookie security SameSite| Pragma| Cache-Control| 
+| medium | low | medium  | low | medium  | low | - | low| low| high| high|
+
+| Edge mobile | | | | | | | |
+| ------------- | - | - | - | - | - | - | -|
+| X-XSS-Protection| CSP | X-Content-Type-Options| X-Frame-Options| Strict-Transport-Security | Cookie security HttpOnly|Pragma| Cache-Control| 
+| medium | low| medium | low| medium  | low | high | high|
+
+| Firefox for Android | | | | | | | | | |
+| ------------- | - | - | - | - | - | - | -| -| -|
+| X-XSS-Protection| CSP | X-Content-Type-Options| X-Frame-Options| Strict-Transport-Security | Public Key Pinning| Referrer-Policy|Cookie security HttpOnly| Pragma| Cache-Control| 
+| medium | low | medium | low| medium  | low | - | low| high| high|
+
+| Internet Explorer | | | | |
+| ------------- | - | - | - | - | 
+| X-Frame-Options | Strict-Transport-Security |Cookie security HttpOnly| Pragma| Cache-Control| 
+| medium | high | medium  | high | high | 
+
+| iOS Safari | | | | | | |
```

Comment:
these are browsers, but what about mobile clients? (urlsession, okhttpclient)

Reply:
not really feasible given the time left